rllib/agents/mbmpo/utils.py

```diff
@@ -67,60 +61,3 @@ def calculate_gae_advantages(paths, discount, gae_lambda):
             path_baselines[:-1]
         path["advantages"] = discount_cumsum(deltas, discount * gae_lambda)
     return paths
-
-
-def discount_cumsum(x, discount):
-    """
-        Returns:
-            (float) : y[t] - discount*y[t+1] = x[t] or rev(y)[t]
-            - discount*rev(y)[t-1] = rev(x)[t]
-        """
-    return scipy.signal.lfilter(
-        [1], [1, float(-discount)], x[::-1], axis=0)[::-1]
-
-
-class MBMPOExploration(Exploration):
```

Comment:
It's not exactly the same as before right? (After moving this to Stochasticsampling)

Reply:
Yes, it hopefully is. :) I just moved the random functionality into StochasticSampling, so that other algos can profit from that as well. The only thing that MBMPOExploration adds now, is setting random_timesteps to 0 for workers-indices > 0.

Reply:
I fixed ignoring the worker-index (it should now be exactly the same as before + cleaned up).